src/libMessage/Messenger.cpp

```diff
@@ -1938,6 +1938,41 @@ bool Messenger::SetAccountStoreDelta(vector<unsigned char>& dst,
   return SerializeToArray(result, dst, offset);
 }
 
+bool Messenger::StateDeltaToAddressMap(
+    const vector<unsigned char>& src, const unsigned int offset,
+    unordered_map<Address, int>& accountMap) {
```

Comment:
int is too small for this case

Reply:
@KaustubhShamshery Can you fix this issue now?